content/attacks/posts/2023-01-12-Lendhub.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2023-01-12
+target-entities: Lendhub
+entity-types: Defi
+attack-types: Smart Contract Exploit
+title: "Lendhub Hacked for $6 Million"
+---
+
+## Summary
+
+A hacker exploited a vulnerability in the LendHub protocol to steal approximately $6 million in digital assets. The vulnerability was caused by the existence of two IBSV tokens on the platform, one of which had been deprecated but not removed. The attacker was able to mint and redeem tokens in the old market while borrowing against them in the new market, ultimately making off with the majority of the assets on the platform.
+
+## Attackers
+
+The attackers behind the Lendhub hack remain unidentified.
+
+- Lendhub Hacker wallet: [0x9d0163e76bbcf776001e639d65f573949a53ab03](https://etherscan.io/address/0x9d0163e76bbcf776001e639d65f573949a53ab03)
+
+## Losses
+
+Lendhub lost approximately $6 million.
+
+## Timeline
+
+- **January 12, 2023:**  Lendhub is hacked
+- **January 13, 2023:**  [Lendhub announced the hack on their platform and the extent of the losses, with around $6 million stolen](https://twitter.com/LendHubDefi/status/1613846541651030018)
+- **January 13, 2023:**  [The attacker has made 11 transactions totaling 1,100 ETH to TornadoCash](https://twitter.com/SlowMist_Team/status/1613906600279900162)
+- **February 26, 2023:**  [The attacker has deposited another 2,415.4 ETH into Tornado.cash](https://twitter.com/PeckShieldAlert/status/1630027178426642432)
+
+## Security Failure Causes
+
+The main reason for the Lendhub attack was the failure to remove a deprecated token from the market properly. This vulnerability allowed attackers to exploit the protocol's smart contracts and steal funds.
```

Comment:
Make bullet point as in other articles, provide little more details and shorten or remove last sentence